modules/virt-about-reclaiming-statically-provisioned-persistent-volumes.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reusing-persistent-volumes.adoc
+
+[id="virt-about-reclaiming-statically-provisioned-persistent-volumes_{context}"]
+
+= About reclaiming statically provisioned persistent volumes
+
+When you reclaim a persistent volume (PV), you unbind the PV from a persistent volume claim (PVC) and delete the PV. Depending on the underlying storage, you might need to manually delete the shared storage.
+
+You can then re-use the PV configuration to create a new PV with a different name.
+
+Statically provisioned PVs should have a reclaim policy of `Retain` to be reclaimed.
```

Comment:
I struggled with this one because it's not imperative. Since we're deleting all resources in the procedure anyway, it didn't seem to fuss the cluster I was testing on.

Reply:
Changing to "must". It might not be strictly true but it's best practice according to k8s docs.